modal/stub.py

```diff
@@ -540,12 +540,23 @@ def _add_function(self, function: _Function, mounts: List[_Mount]) -> _FunctionH
 
     @property
     def registered_functions(self) -> List[str]:
+        """Names of modal.Function objects registered on the stub."""
         return list(self._function_handles.keys())
 
     @property
     def registered_entrypoints(self) -> Dict[str, Callable]:
+        """Names of local CLI entrypoints registered on the stub."""
         return self._local_entrypoints
 
+    @property
+    def registered_endpoints(self) -> List[str]:
```

Comment:
nit: would be nice with a slightly more distinct name from entrypoints, perhaps include the word `web`